### ISSUE TITLE
More verbose output at setup

### ISSUE THREAD

**Issue by koppor** [ilammy/msvc-dev-cmd#15]
The current output of the setup is as follows:

```
Run ilammy/msvc-dev-cmd@v1
  with:
    sdk: 8.1
    toolset: 14.11
    arch: x64
Configured Developer Command Prompt
```

My compiler has following output:

```
 C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Platforms\Win32\PlatformToolsets\v140\Toolset.targets(34,5): error MSB8036: The Windows SDK version 8.1 was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution". [D:\a\clink\clink\.build\vs2015\clink_app_common.vcxproj]
```

Not sure how to proceed here. The full GitHub action workflow is available at https://github.com/koppor/clink/pull/2.

**Comment by koppor** [comment]
Maybe, it's because of some Visual Studio 2015 issue (refs #7). With VS 2019, it works:

```yaml
      - uses: ilammy/msvc-dev-cmd@v1
        with:
          sdk: "8.1"
          toolset: "14.16"
```

**Comment by ilammy** [comment]
Hm... First of all, for avoidance of doubt, this action *does not* install Visual Studio or toolchains, it only locates the installed ones. If the combination or SDK/toolset/architecture is not available on the runner then it can't be enabled using `msvc-dev-cmd`. 

Windows runners provided by GitHub do not include VS2015. Only VS2019 (in `windows-2019` aka `windows-latest`) and VS2017 (in `windows-2016`) are currently available. If you need older versions then you'd need to either use your own runners where it is preinstalled (it seems to be kinda popular), or somehow install VS or missing components during the build.

**Comment by koppor** [comment]
> Hm... First of all, for avoidance of doubt, this action _does not_ install Visual Studio or toolchains, it only locates the installed ones.

Maybe the determined location could be output. Thus, one could (indirectly) learn that something is not available.

**Comment by ilammy** [comment]
Okay, this finally has been published in v1.5.0 (also available as the default v1) 🎉
